lib/console_manager.rb

```diff
@@ -124,8 +126,53 @@ def list_all_music
   # list all movies
 
   # add a game
+  def add_game
+    # Prompt
+    puts
+    puts 'Please provide the details of the game.'
+    print 'Title: '
+    title = gets.chomp
+    print 'Multiplayer (Y/N): '
+    multiplayer = gets.chomp.upcase
+    print 'Last Played Date (YYYY/MM/DD): '
+    last_played_date = gets.chomp
+    print 'Publish Date (YYYY/MM/DD): '
+    publish_date = gets.chomp
+
+    # Input validation
+    date_pattern = %r{\A\d{4}/\d{2}/\d{2}\z}
+    until publish_date.match?(date_pattern)
+      puts "\nPlease enter the date in this format: YYYY/MM/DD"
+      publish_date = gets.chomp
+    end
+
+    # Parse dates into Date objects
+    last_played_at = Date.parse(last_played_date)
+    publish_date = Date.parse(publish_date)
+
+    print 'Archived (Y/N): '
+    archived = gets.chomp.upcase
+
+    # Action
+    @game_manager.add_game(title, multiplayer == 'Y', last_played_at, publish_date, archived == 'Y')
```

Comment:
I think you should give the user the option to add an **Author** to the **Game** they just created. Currently, there is no way a user can add an author. You can refer to my implementation that lets the user add a **Label** to a **Book**.